src/Protocol/Http/Authenticator/JwtAuthenticator.php

```diff
@@ -40,14 +55,37 @@ public function __construct(JwtInterface $jwt)
         $this->jwt = $jwt;
     }
 
+    /**
+     * Get jws lifetime
+     *
+     * @return integer
+    */
+    public function getJwsLifetime() : int
+    {
+        return $this->jwsLifetime;
+    }
+
+    /**
+     * Set jws lifetime
+     *
+     * @param integer $jwsLifetime
+     */
+    public function setJwsLifetime(int  $jwsLifetime)
+    {
+        $this->jwsLifetime = abs($jwsLifetime);//ignore sign
```

Comment:
I think, what better we should throw exception on invalid value because the developer can not understand what he set the negative value, but the application correct work.

Note: I try to write this library with the full strict mode. Invalid value - exception.

Thank for understand.